eng/deploy.yaml

```diff
@@ -102,9 +102,14 @@ stages:
     - download: current
       artifact: Maestro.Data
     - task: UseDotNet@2
-      displayName: Use .NET Core SDK
+      displayName: Install Correct .NET Version
       inputs:
-        packageType: sdk
+        useGlobalJson: true
+
+    - task: UseDotNet@2
```

Comment:
Does this mean that something in the solution still requires the 3.1 runtime?

Reply:
Honestly, I didn't want to remove things, so I was just adding more.  They _probably_ aren't necessary, but I really just want this to work, because main is broken, so we can try to clean it up a bit later once I can get a clean deployment.

Reply:
I bit the bullet and removed it. Lets see what happens!